lighthouse/datadog_checks/lighthouse/lighthouse.py

```diff
@@ -0,0 +1,83 @@
+# the following try/except block will make the custom check compatible with any Agent version
+try:
```

Comment:
You could directly use 
```python
from datadog_checks.base import AgentCheck
```
as there is already a similar compatibility layer.

Reply:
will update, ty